### PR TITLE
Remove template structure by language (#1785)

### PR DESCRIPTION
**Pull request by vlo-rte**

### CODE REVIEW

**Comment by freddidierRTE** [comment]
Clean related code in UI , at least : 
- in method fetchHbsTemplate ui/main/src/app/services/processes.service.ts
- in method  queryTemplate in ui/main/src/app/modules/cards/services/handlebars.service.ts

File: ui/main/src/app/services/processes.service.spec.ts [hunk 1]
```diff
@@ -72,20 +72,15 @@ describe('Processes Services', () => {
 
 
     describe('#fetchHbsTemplate', () => {
-            const templates = {
-                en: 'English template {{card.data.name}}',
-                fr: 'Template Français {{card.data.name}}'
-            };
+            const template = 'template {{card.data.name}}';
         it('should return different files for each language', () => {
```
Comment (freddidierRTE) [inline ui/main/src/app/services/processes.service.spec.ts:76]: Title of test is not valid anymore .

Anyway , remove it as it is not useful

**Review by freddidierRTE — CHANGES_REQUESTED** [review]

File: ui/main/src/app/services/processes.service.spec.ts (hunk 1 above)
Comment (vlo-rte) [reply ui/main/src/app/services/processes.service.spec.ts]: Done

**Comment by vlo-rte** [comment]
> Clean related code in UI , at least :
> 
>     * in method fetchHbsTemplate ui/main/src/app/services/processes.service.ts
> 
>     * in method  queryTemplate in ui/main/src/app/modules/cards/services/handlebars.service.ts

Done.

**Pull request merged**